### PR TITLE
Adds some missing migration steps, and fixes a typo

### PR DESCRIPTION
**Pull request by darrellwarde**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
Looks like you've updated the documentation!

Check out your changes at https://neo4j-docs-graphql-25.surge.sh

File: modules/ROOT/pages/migration/v4-migration/index.adoc [hunk 1]
```diff
@@ -572,6 +572,20 @@ type query {
 
 Additionally, escaping strings is no longer needed.
 
+=== `@exclude` removed
```
Comment (lidiazuin) [inline modules/ROOT/pages/migration/v4-migration/index.adoc:575]: Can't we use the label deprecated for it?

```suggestion
=== `@exclude` label:[deprecated]
```

File: modules/ROOT/pages/migration/v4-migration/index.adoc [hunk 2]
```diff
@@ -572,6 +572,20 @@ type query {
...
+
+The `@exclude` directive has been removed in favor of much more granular configuration directives.
```
Comment (lidiazuin) [inline modules/ROOT/pages/migration/v4-migration/index.adoc:577]:
```suggestion
The `@exclude` directive has been removed in favor of a more granular configuration directives.
```

File: modules/ROOT/pages/migration/v4-migration/index.adoc [hunk 3]
```diff
@@ -572,6 +572,20 @@ type query {
...
+
```
Comment (lidiazuin) [inline modules/ROOT/pages/migration/v4-migration/index.adoc:578]:
```suggestion
```

File: modules/ROOT/pages/migration/v4-migration/index.adoc [hunk 4]
```diff
@@ -572,6 +572,20 @@ type query {
...
+The new `@query`, `@mutation` and `@subscription` directives instead allow for fully granular configuration for each operation.
```
Comment (lidiazuin) [inline modules/ROOT/pages/migration/v4-migration/index.adoc:579]:
```suggestion
The `@query`, `@mutation` and `@subscription` directives instead allow for fully granular configuration for each operation.
```

File: modules/ROOT/pages/migration/v4-migration/index.adoc [hunk 5]
```diff
@@ -572,6 +572,20 @@ type query {
...
+
+As a direct migration, the following usages are equivalent:
+
+* `@exclude` and `@query(read: false, aggregate: false) @mutation(operations: []) @subscription(events: [])`.
+* `@exclude(operations: [READ])` and `@query(read: false, aggregate: false)`.
+* `@exclude(operation: [CREATE, UPDATE, DELETE])` and `@mutation(operations: [])`.
+
+Whilst there is more verbosity, the directives are significantly more powerful and extensible as the library gains features.
```
Comment (lidiazuin) [inline modules/ROOT/pages/migration/v4-migration/index.adoc:587]:
```suggestion
Despite the verbosity, the directives are significantly more powerful and extensible as the library gains features.
```

File: modules/ROOT/pages/migration/v4-migration/index.adoc [hunk 6]
```diff
@@ -854,6 +868,16 @@ type Record @limit(default: 10, max: 100) {
 }
 ----
 
+=== `@readonly` and `@writeonly` removed
+
+The `@readonly` and `@writeonly` directives have been removed in favor of more granular configuration directives.
+The new `@selectable` and `@settable` directives can be used to configure not only if fields are readable or writable, but also when they should be readable or writable.
```
Comment (lidiazuin) [inline modules/ROOT/pages/migration/v4-migration/index.adoc:874]:
```suggestion
The `@selectable` and `@settable` directives can be used to configure not only if fields are readable or writable, but also when they should be readable or writable.
```

File: modules/ROOT/pages/migration/v4-migration/index.adoc [hunk 7]
```diff
@@ -854,6 +868,16 @@ type Record @limit(default: 10, max: 100) {
 }
 ----
 
+=== `@readonly` and `@writeonly` removed
```
Comment (lidiazuin) [inline modules/ROOT/pages/migration/v4-migration/index.adoc:871]: Same here about the label

**Review by lidiazuin — COMMENTED** [review]
I've seen you merged the PR while I was reviewing, so I'll just leave these comments to catch up on a new PR for an editorial review :)

**Pull request merged**